interfaces/builtin/browser_support.go

```diff
@@ -385,6 +392,15 @@ func (iface *browserSupportInterface) AppArmorConnectedPlug(spec *apparmor.Speci
 	spec.AddSnippet(browserSupportConnectedPlugAppArmor)
 	if allowSandbox {
 		spec.AddSnippet(browserSupportConnectedPlugAppArmorWithSandbox)
+		// if apparmor supports userns mediation then add this too
+		if apparmor_sandbox.ProbedLevel() != apparmor_sandbox.Unsupported {
+			if features, err := apparmor_sandbox.ParserFeatures(); err == nil {
```

Comment:
wouldn't be more consistent to report possible errors here and similar places?

Reply:
I also wonder if we should have some dedicated helper for this kind of code but that's orthogonal to this PR

Reply:
Regarding reporting errors - this feature will not be available in the apparmor parser on older Ubuntu releases BUT as the kernel on those releases is unlikely to support it either then it doesn't matter that the parser doesn't support it - since the kernel will not block userns. So it is not fatal - and would likely just be noise on these older releases etc. So I had intentionally left this out as I see this as best-effort rather than required.

Reply:
but ParserFeatures is not meant to error on just missing features, no? it errors on more serious problems?  my question is specifically about err != nil. Compare with the similar preexisting logic in networkControlInterface.AppArmorConnectedPlug

Reply:
Ah okay, thanks for explaining it to me - yep I agree, added in 5ce0f416d9